docs/minikube.md

```diff
@@ -26,7 +26,7 @@ minikube start --cni=antrea.yml --network-plugin=cni
 
 These instructions assume that you have built the Antrea Docker image locally
 (e.g. by running `make` from the root of the repository, or in case of arm64 architecture by running
-`DOCKER_BUILDKIT=1 ./hack/build-antrea-ubuntu-all.sh --platform linux/arm64`).
+`./hack/build-antrea-ubuntu-all.sh --platform linux/arm64`).
```

Comment:
since you are modifying this line, the current name of the script is `./hack/build-antrea-linux-all.sh`, it's not `./hack/build-antrea-ubuntu-all.sh` anymore :)

Reply:
Updated, thanks:)